src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -547,7 +547,10 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="extractFunction" value="Cmd+Alt+X"/>
          <shortcut refid="extractLocalVariable" value="Cmd+Alt+V"/>
          <shortcut refid="commentUncomment" value="Cmd+Shift+C"/>
-         <shortcut refid="reindent" value="Cmd+I"/>
+         <shortcut refid="blockOutdent" value="Meta+[" if="org.rstudio.core.client.BrowseCap.isMacintosh()" disableModes="default,vim,emacs"/>
+         <shortcut refid="blockIndent" value="Meta+]" if="org.rstudio.core.client.BrowseCap.isMacintosh()" disableModes="default,vim,emacs"/>
+         <shortcut refid="reindent" value="Cmd+I" disableModes="default,vim"/>
+         <shortcut refid="reindent" value="Cmd+I" disableModes="default,vim"/>
```

Comment:
Looks like this command entry was duplicated. Also not sure why we're disabling in default + vim modes?

Reply:
I am sorry. I was testing things...will fix it.